### PR TITLE
DOC: Updated doc of nonzero()

### PR DESCRIPTION
**Pull request by jsh9**
Updated documentation of nonzero() to eliminate ambiguity

### CODE REVIEW

**Comment by eric-wieser** [comment]
Commit should start with `DOC: `

**Comment by jsh9** [comment]
I fixed the title of this commit. It now starts with "DOC:"

**Comment by charris** [comment]
You need to change the commit message heading. That will require `git rebase -i HEAD^` followed by a force push. If you need help with that, squawk.

**Comment by jsh9** [comment]
Yes, I need some help.  I am now making changes on the web page end.  (And I am still learning how to use git on terminal.)

**Comment by jsh9** [comment]
So I submitted another commit (just in order to add "DOC:" to the commit message heading). I only added two spaces in this commit.

**Comment by eric-wieser** [comment]
You should `rebase --squash` those two commits together.

What you should have done before you made that second one is `git commit --amend`, which would edit the old commit.

**Comment by eric-wieser** [comment]
>  I am now making changes on the web page end. 

Ah, unfortunately that doesn't allow you to amend the commit. You'll either need to use the terminal, or wait for me or someone else to squash it for you

File: numpy/core/fromnumeric.py [hunk 1]
```diff
@@ -1509,13 +1509,12 @@ def nonzero(a):
...
     >>> x
-    array([[ 1.,  0.,  0.],
-           [ 0.,  1.,  0.],
-           [ 0.,  0.,  1.]])
-    >>> np.nonzero(x)
-    (array([0, 1, 2]), array([0, 1, 2]))
+    array([[1, 0, 0],
+           [0, 2, 0],
+           [1, 1, 0]])
+    >>> (array([0, 1, 2, 2], dtype=int64), array([0, 1, 0, 1], dtype=int64))
```
Comment (eric-wieser) [inline numpy/core/fromnumeric.py]: Where did `>>> np.nonzero(x)` go?

**Pull request merged**